### ISSUE TITLE
[Problem]VS version

### ISSUE THREAD

**Issue by JoeyCrane** [IntelVCL/Open3D-PointNet2-Semantic3D#43]
I am with VS2017+cuda9.0+cmake3.14, my cmake puts an error.
"Selecting Windows SDK version 10.0.17763.0 to target Windows 10.0.17134.
The CXX compiler identification is MSVC 19.16.27024.1
The CUDA compiler identification is unknown
Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.16.27023/bin/Hostx86/x64/cl.exe
Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.16.27023/bin/Hostx86/x64/cl.exe -- works
Detecting CXX compiler ABI info
Detecting CXX compiler ABI info - done
Detecting CXX compile features
Detecting CXX compile features - done
CMake Error at CMakeLists.txt:5 (project):
  No CMAKE_CUDA_COMPILER could be found.
Configuring incomplete, errors occurred!"

My cmakeError.log shows details :" C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.16.27023\include\type_traits(603): **error : expression must have a constant value** [D:\pointcloud\Open3D-PointNet2-Semantic3D-master\Open3D-PointNet2-Semantic3D-master\tf_ops\build\CMakeFiles\3.14.0-rc2\CompilerIdCUDA\CompilerIdCUDA.vcxproj]"

And if I compile CompilerIdCUDA.vcxproj with vs2017 but choose vs2015(v140)'s platform toolset, it can be pass successfully without errors.Does that mean I should download vs2015?

**Comment by yxlao** [comment]
Yeah, the CUDA + TensorFlow part is not tested on Windows, please use Ubuntu instead
